### PR TITLE
OCPBUGS-9959: check scheduler settings under /sys/kernel/debug/sched/ 

### PR DESCRIPTION
**Pull request by mrniranjan**
…sctl

On RHEL9.2 scheduler settings have been moved under  /sys/kernel/debug/sched/ instead of setting them using sysctl

### CODE REVIEW

File: test/e2e/performanceprofile/functests/1_performance/performance.go [hunk 1]
```diff
@@ -1352,6 +1355,21 @@ func execSysctlOnWorkers(workerNodes []corev1.Node, sysctlMap map[string]string)
...
 
+// check scheduler settings. on RHCOS9.2 all scheduler settings are moved to /sys/kernel/debug/sched/
+func checkSchedKnobs(workerNodes []corev1.Node, schedKnobs map[string]string) {
+	var err error
+	var out []byte
+	for _, node := range workerNodes {
+		for param, expected := range schedKnobs {
+			By(fmt.Sprintf("Checking scheduler knob %s", param))
+			knob := fmt.Sprintf("/rootfs/sys/kernel/debug/sched/%s", param)
+			out, err = nodes.ExecCommandOnMachineConfigDaemon(&node, []string{"cat", knob})
```
Comment (ffromani) [inline test/e2e/performanceprofile/functests/1_performance/performance.go:1366]: I would have made this loop variables but no big deal

**Review by ffromani — COMMENTED** [review]
/lgtm
/approve

**Comment by Tal-or** [comment]
this is the CPU load balancing which is fallen on rhel 9.2 I suggest to override the lane
```
[FAILED] in [It] - /go/src/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/1_performance/cpu_management.go:310 @ 03/26/23 17:30:59.865
• [FAILED] [1422.792 seconds]
[rfe_id:27363][performance] CPU Management when pod runs with the CPU load balancing runtime class [It] [test_id:32646] should disable CPU load balancing for CPU's used by the pod
/go/src/github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/1_performance/cpu_management.go:302
  [FAILED] Unexpected error:
      <*errors.errorString | 0xc0002cf0e0>: {
          s: "timed out waiting for the condition",
      }
      timed out waiting for the condition
```

**Comment by yanirq** [comment]
@mrniranjan  please remove the skip in the test (OCPBUGS-9959) so the PR can be properly tested

**Comment by mrniranjan** [comment]
@yanirq there is no skip in this PR

**Comment by yanirq** [comment]
> @yanirq there is no skip in this PR

you might have to rebase and check then

**Comment by ffromani** [comment]
should probably be rebased on top of main branch

**Comment by yanirq** [comment]
> should probably be rebased on top of main branch

/cc @mrniranjan

**Comment by yanirq** [comment]
@mrniranjan  please remove `testutils.KnownIssueJira("OCPBUGS-9959")` occurrences  so the test will be applied

**Comment by ffromani** [comment]
thanks @mrniranjan

**Comment by ffromani** [comment]
/lgtm cancel

per https://github.com/openshift/cluster-node-tuning-operator/pull/581#issuecomment-1495499511

**Comment by yanirq** [comment]
@mrniranjan  you will have to align the rendered output for the render test (manual_tuned.yaml) due to the changes of the base file

**Pull request merged**